### PR TITLE
ft/air-page

### PR DESCRIPTION
**Pull request by kelvinkipruto**
## Description
This PR Updates the Air Page with new content and cities

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -331,3 +331,4 @@ $RECYCLE.BIN/
 .now
 
 # End of Custom
+.yarn
```
Comment (kilemensi) [inline .gitignore:334]: ?

File: src/components/SearchBar.js [hunk 2]
```diff
@@ -294,8 +294,12 @@ const DEFAULT_OPTIONS = [
   { value: "nairobi", label: "Nairobi, Kenya" },
   { value: "kisumu", label: "Kisumu, Kenya" },
   { value: "nakuru", label: "Nakuru, Kenya" },
-  { value: "lagos", label: "Lagos, Nigeria" },
   { value: "dar-es-salaam", label: "Dar-es-Salaam, Tanzania" },
+  { value: "abuja", label: "Abuja, Nigeria" },
+  { value: "lagos", label: "Lagos, Nigeria" },
+  { value: "port-harcourt", label: "Port Harcourt, Nigeria" },
+  { value: "ilorin", label: "Ilorin, Nigeria" },
+  { value: "maiduguri", label: "Maiduguri, Nigeria" },
```
Comment (kilemensi) [inline src/components/SearchBar.js:302]: Sort cities in a given country alphabetically... And may be group them by country if it's [simple](https://react-select.com/advanced#replacing-builtins)

**Review by kilemensi — COMMENTED** [review]
👍🏽

File: .gitignore (hunk 1 above)
Comment (kilemensi) [reply .gitignore:334]: Shouldn't. Are you using [`yarn classic`](https://classic.yarnpkg.com/lang/en/)?

Comment (kelvinkipruto) [reply .gitignore:334]: yarn 3

**Comment by kelvinkipruto** [comment]
@kilemensi This is ready for review

File: src/components/PartnerLogos.js [hunk 3]
```diff
@@ -2,7 +2,10 @@ import { Grid, Typography } from "@material-ui/core";
 import { makeStyles } from "@material-ui/core/styles";
 import React from "react";
 
+import car from "@/sensorsafrica/assets/images/partners/car.png";
+import godown from "@/sensorsafrica/assets/images/partners/godown.png";
```
Comment (kilemensi) [inline src/components/PartnerLogos.js:6]: Can we ask for a version with white background and black text? If not, try inverting it in any photo editing software.

File: src/components/PartnerLogos.js [hunk 4]
```diff
@@ -2,7 +2,10 @@ import { Grid, Typography } from "@material-ui/core";
 import { makeStyles } from "@material-ui/core/styles";
 import React from "react";
 
+import car from "@/sensorsafrica/assets/images/partners/car.png";
```
Comment (kilemensi) [inline src/components/PartnerLogos.js:5]: We need a higher quality one.

File: src/components/PartnerLogos.js [hunk 5]
```diff
@@ -97,6 +100,42 @@ function PartnerLogos(props) {
...
+
+      <Grid
+        container
+        direction="row"
+        justifyContent="center"
+        alignItems="center"
+        className={classes.partnersLogo}
+      >
+        <Grid item xs={12} sm={4} md={3}>
+          <div className={classes.worldBankLogo}>
```
Comment (kilemensi) [inline src/components/PartnerLogos.js:112]: worldBankLogo?

File: src/components/SearchBar.js [hunk 6]
```diff
@@ -291,17 +305,55 @@ const components = {
...
 
 const DEFAULT_OPTIONS = [
-  { value: "nairobi", label: "Nairobi, Kenya" },
-  { value: "kisumu", label: "Kisumu, Kenya" },
-  { value: "nakuru", label: "Nakuru, Kenya" },
-  { value: "lagos", label: "Lagos, Nigeria" },
-  { value: "dar-es-salaam", label: "Dar-es-Salaam, Tanzania" },
+  { value: "nairobi", label: "Nairobi", country: "Kenya" },
+  { value: "kisumu", label: "Kisumu", country: "Kenya" },
+  { value: "nakuru", label: "Nakuru", country: "Kenya" },
```
Comment (kilemensi) [inline src/components/SearchBar.js:310]: Sort them alphabetically.

**Review by kilemensi — COMMENTED** [review]
👍🏽 ... Looks good.

File: src/components/SearchBar.js [hunk 7]
```diff
@@ -291,17 +305,55 @@ const components = {
...
+    const { country } = city;
+    if (!acc[country]) {
+      acc[country] = [];
+    }
+    acc[country].push(city);
+    return acc;
+  }, {});
+
+  // create options for each country
+  const countryOptionsWithCities = countryOptions.map((option) => {
```
Comment (kilemensi) [inline src/components/SearchBar.js:349]:
```suggestion
  const countryOptionsWithCities = Object.keys(groupedCities).map((country) => {
```
I think the above will contain unique countries already without the need of `countries` or `countryOptions` above (`if (!acc[country]) {` ensures we don't duplicate `country` key).

File: package.json [hunk 8]
```diff
@@ -82,5 +82,9 @@
     "*.yml": [
       "yarn format-staged"
     ]
+  },
+  "volta": {
+    "node": "16.18.1",
+    "yarn": "1.22.19"
   }
-}
+}
```
Comment (kilemensi) [inline package.json:90]:
```suggestion
}

```

**Review by kilemensi — APPROVED** [review]
🚀

File: src/components/SearchBar.js (hunk 7 above)
Comment (kelvinkipruto) [reply src/components/SearchBar.js:349]: Awesome

**Pull request merged**